src/StreamFactoryInterface.php

```diff
@@ -48,5 +16,5 @@ public function createStreamFromResource($body);
      *
      * @return StreamInterface
      */
-    public function createStreamFromString($body);
+    public function createStream($body = '');
```

Comment:
Diactoros allows for a string or a resource. [Slim](https://github.com/slimphp/Slim/blob/1dedd2592c4f7e088ee3a87a7936f5df71cb7593/Slim/Http/Stream.php#L96) only allows a resource, but an implemented factory could easily create create a resource from a provided string.